Multisig/Data/Services/Safe Client Gateway Service/BalancesRequest.swift

```diff
@@ -35,8 +34,8 @@ struct BalancesRequest: JSONRequest {
 }
 
 extension BalancesRequest {
-    init(_ address: Address) {
-        self.init(safeAddress: address.checksummed, fiat: AppSettings.selectedFiatCode)
+    init(_ address: Address, chainId: Int) {
```

Comment:
As a next refactoring step (our of scope of this PR) I would look into passing everywhere Safe object, that already has it's network, so probably we can remove network binding logic

Reply:
@sche hm, maybe that's not a good one, because this will couple the request to the database model, I think the current approach is good